### PR TITLE
Clesaec/tdi 44195 sonar analyse

### PR DESCRIPTION
**Pull request by clesaec**

### CODE REVIEW

**Review by rlecomte-talend — DISMISSED** [review]
LGTM

**Review by undx — COMMENTED** [review]
We have a two distinct purposes/goals in that PR, please split it in two different PRs.
That's more pertinent and in the case we've to revert...

**Review by undx — CHANGES_REQUESTED** [review]
See previous comment.

File: Jenkinsfile [hunk 1]
```diff
@@ -83,6 +86,7 @@ spec:
         choice(name: 'Action', 
                choices: [ 'STANDARD', 'PUSH_TO_XTM', 'DEPLOY_FROM_XTM', 'RELEASE' ],
                description: 'Kind of running : \nSTANDARD (default), normal building\n PUSH_TO_XTM : Export the project i18n resources to Xtm to be translated. This action can be performed from master or maintenance branches only. \nDEPLOY_FROM_XTM: Download and deploy i18n resources from Xtm to nexus for this branch.\nRELEASE : build release')
+        booleanParam(name: 'FORCE_SONAR', defaultValue: false, description: 'Turn to true to force sonar analysis')
```
Comment (undx) [inline Jenkinsfile:89]: Rather say: _Force Sonar analysis_

**Review by undx — APPROVED** [review]
LGTM 
Just change label as commented.

**Review by undx — APPROVED** [review]
LGTM

**Pull request merged**